assets/app/styles/_sidebar.less

```diff
@@ -249,6 +254,7 @@
           .dl-horizontal {
             dt {
               float: none;
+              width: 100%;
```

Comment:
if this is undoing `width: (@component-offset-horizontal - 20);`, would `width: auto` be more correct?